src/main/java/com/hladchenko/springrestapi/controller/UserController.java

```diff
@@ -0,0 +1,60 @@
+package com.hladchenko.springrestapi.controller;
+
+import com.hladchenko.springrestapi.entity.Note;
+import com.hladchenko.springrestapi.entity.User;
+import com.hladchenko.springrestapi.service.UserService;
+import org.springframework.hateoas.EntityModel;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.linkTo;
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.methodOn;
+
+@RequestMapping("/user")
+@RestController
+public class UserController {
+    UserService userService;
```

Comment:
why don't you make it as _private_?

Reply:
Agree, will change it to private. Thanks!